test/server/test_fxa_auth.py

```diff
@@ -0,0 +1,78 @@
+import urlparse
+import urllib
+from clientlib import ScreenshotsClient
+from urlparse import urljoin
+import random
+import requests
+import time
+from requests import HTTPError
+
+# Hack to make this predictable:
+random.seed(0)
+
+
+def get_url(url, expect=None):
+    result = urllib.urlopen(url)
+    if expect and result.getcode() != expect:
+        raise Exception("GET {} returned status {}".format(url, result.getcode()))
+    return result.read()
+
+# test we can change expiration and delete shots made previously
+def test_owned_shot_with_fxa_auth():
+    first_user = ScreenshotsClient()
+    second_user = ScreenshotsClient()
+
+    first_user.login();
+    shot_url = first_user.create_shot(docTitle="A_TEST_SITE_1")
+    shot_id = urlparse.urlsplit(shot_url).path.strip("/")
+    shot_page = first_user.read_shot(shot_url)
+    print(first_user.read_shot(shot_url)["clip_url"], shot_page["clip_url"])
+    assert first_user.read_shot(shot_url)["clip_content"] == shot_page["clip_content"]
+    assert "A_TEST_SITE_1" in shot_page["page"]
```

Comment:
You don't really need to test that the shot is uploaded correctly, we have other tests for that.

Reply:
Removed shot upload checks related assert